src/three/plugins/GoogleCloudAuthPlugin.js

```diff
@@ -84,20 +110,16 @@ export class GoogleCloudAuthPlugin {
 
 	getAttributions( target ) {
 
-		if ( this._attribution.value ) {
+		if ( this.logoUrl ) {
 
-			if ( this.logoUrl ) {
-
-				this._logoAttribution.value = this.logoUrl;
-				target.push( this._logoAttribution );
-
-			}
-
-			this._attribution.value = this._attributionsManager.toString();
-			target.push( this._attribution );
+			this._logoAttribution.value = this.logoUrl;
+			target.push( this._logoAttribution );
 
 		}
 
+		this._attribution.value = this._attributionsManager.toString();
+		target.push( this._attribution );
```

Comment:
It looks like there I miscommitted something previously - rather than checking the attribution string we should check the number of visible tiles - see [this condition in master](https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/master/src/three/plugins/GoogleCloudAuthPlugin.js#L86). I'd prefer to keep it that way so we don't return any attributions if there are no tiles to display attributions for.